expected-consensus.md

```diff
@@ -324,9 +324,23 @@ Due to network propagation delay, it is possible for a miner in round N+1 to omi
 
 It is possible for forks to emerge naturally in Expected Consensus. EC relies on weighted chains in order to quickly converge on 'one true chain', with every block adding to the chain's weight. This means the heaviest chain should reflect the most amount of work performed, or in Filecoin's case, the most storage provided.
 
-The weight at each block is equal to its `ParentWeight`, plus that block's delta weight. Delta
-weight is a constant `V`, plus `X` - a function of the total power in the network as reported in the Power Table.  The exact value for `V` and the magnitude of the power ratio value are
-still to be determined, but for now `V = 10` and `X = log(TotalPower)`.
+The weight at each block is equal to its `ParentWeight` plus that block's delta weight. Delta
+weight is a term P times the sum of a constant `V` times the number of blocks included in that TipSet in the given round (can only be calculated in the next round), and `X` - a function of the total power in the network as reported in the Power Table. P will be equal to some term P_i^P_n and seeks to punish selfish mining. So we have:
+
+`Weight = ParentWeight + P(V*NumBlocksMinedInThisRound + X)`
+
+The weight should be rounded down to its fourth decimal at each height (i.e. each time it is recalculated).
+
+```sh
+Note that if your implementation does not allow for rounding to the fourth decimal, miners should apply the [tie-breaker below](#selecting-between-tipsets-with-equal-weight). Weight changes will be on the order of single digit numbers on expectation, so this should not have an outsized impact on chain consensus across implementations.
+```
+
+ The exact value for these parameters remain to be determined, but for illustration, we can set:
+
+- `V = 2 * number of blocks in the round`
+- `X = log(TotalPower in Power Table l blocks back)`
```

Comment:
just to be clear, is this "blocks" back or "tickets" back?

Reply:
good catch!!!! Rounds (as in tickets) back but either way, L is gone